connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -197,6 +200,32 @@
             }
         }
 
+        final Object keepAliveStrategy = config.getProperties().get(ApacheClientProperties.KEEPALIVE_STRATEGY);
+        if (keepAliveStrategy != null) {
+            if (!(keepAliveStrategy instanceof ConnectionKeepAliveStrategy)) {
+                LOGGER.log(
+                        Level.WARNING,
+                        LocalizationMessages.IGNORING_VALUE_OF_PROPERTY(
+                                ApacheClientProperties.KEEPALIVE_STRATEGY,
+                                keepAliveStrategy.getClass().getName(),
+                                ConnectionKeepAliveStrategy.class.getName())
+                );
+            }
+        }
+
+        final Object reuseStrategy = config.getProperties().get(ApacheClientProperties.REUSE_STRATEGY);
+        if (reuseStrategy != null) {
+            if (!(reuseStrategy instanceof ConnectionReuseStrategy)) {
+                LOGGER.log(
+                        Level.WARNING,
+                        LocalizationMessages.IGNORING_VALUE_OF_PROPERTY(
```

Comment:
The same thing is valid here: 
the code actually does not ignore value of the property, it only logs intention to do so. And the if inside another if could be rewritten as if (A && !B).

Reply:
Whoops, how silly, I just copied the connectionManager block and didn't change it to null the values if invalid. I'll fix now

Reply:
Ok I've committed the update to the pull